src/main/java/com/metaphorce/common/dtos/UserDTO.java

```diff
@@ -28,4 +30,29 @@ public class UserDTO {
      * The email address of the user.
      */
     private String email;
-}
\ No newline at end of file
+
+    /**
+     * The password of the user.
+     */
+    private String password;
+
+    /**
+     * The date and time when the user was created.
+     */
+    private LocalDateTime creationDate;
+
+    /**
+     * The date and time when the user was last updated.
+     */
+    private LocalDateTime lastUpdate;
+
+    /**
+     * Indicates whether the user is soft deleted.
+     */
+    private boolean softDelete;
```

Comment:
like the best practice you should name your variable with prefix "is" like isenabled ...

Reply:
If we need a complete history of users, including their deletions, and want the ability to restore users, I would recommend using softDelete. On the other hand, if simplicity is key and you only need to know whether a user is active or not, isEnabled might be sufficient.